lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -462,6 +472,10 @@ type DelegationFee = "wallets"
     :> "delegation-fees"
     :> Get '[JSON] ApiFee
 
+type PoolMaintenance = "stake-pools"
+    :> ReqBody '[JSON] ApiMaintenanceAction
```

Comment:
This is missing an intermediate `maintenance-actions` path segment, otherwise a `POST` on `/stake-pools` implies that we are creating a stake pool.